### DISCUSSION TITLE
RD 1.6.0 mac aarch64 is reported as intel app on mac m1 (Monterey).

### DISCUSSION THREAD

**Discussion by nikk0** [rancher-sandbox/rancher-desktop#3227] [General]
Hello.
I just downloaded Rancher.Desktop-1.6.0-mac.aarch64.zip on m M1 mac (Monterey 12.6) but the embedded app Rancher-Desktop.app is seen as type : Application(Intel), ie no m1 native.
Did I miss something ? aarch64 is supposed to be related to arm64 m1 architecture, right ?   
Regards

**Comment by mook-as** [comment]
Hi! At the moment, the UI is still running on Intel (under Rosetta), but the actual VM machinery (qemu) is arm-native.  Please see #1416 for details.

**Reply by nikk0** [reply]
thanks for your feedback!
